sysml.library/Kernel Library/Occurrences.kerml

```diff
@@ -161,19 +167,19 @@ package Occurrences {
 	}
 	
 	/**
-	 * TimeSliceOf asserts once occurrence is a time slice of another.
+	 * TimeSliceOf asserts once occurrence is a time slice of another, including at least itself.
 	 */
 	assoc TimeSliceOf {
 		end feature timeSliceOf: Occurrence[1..*] redefines Occurrence::timeSliceOf, BinaryLink::source;
-		end portion feature timeSlice: Occurrence[0..*] redefines Occurrence::timeSlices, BinaryLink::target;
+		end portion feature timeSlice: Occurrence[1..*] redefines Occurrence::timeSlices, BinaryLink::target;
 	}
 	
 	/**
 	 * SnapshotsOf asserts once occurrence is a snapshot of another.
 	 */
 	assoc SnapshotOf {
```

Comment:
Same comment as above applies to snapshots, they should specialize HappensDuring:
  assoc SnapshotOf specializes HappensDuring.
Can file a separate issue if you want.

Reply:
Shouldn't `SnapshotOf` specialize `TimeSliceOf`?